src/module/skb/bpf/skb_hook.bpf.c

```diff
@@ -314,6 +314,14 @@ DEFINE_HOOK(F_AND, RETIS_F_PACKET_PASS,
 	struct sk_buff *skb;
 
 	skb = retis_get_sk_buff(ctx);
+	if (!skb) {
+		struct nft_traceinfo *info = retis_get_nft_traceinfo(ctx);
```

Comment:
I'm thinking this might not be the only place where the skb is "hidden" behind some other struct. Do you think it makes sense to refactor this into common.h?

Reply:
Totally makes sense. 
I added a new helper to embed this logic (as it might change in the future):

`static __always_inline struct sk_buff *retis_get_skb(struct retis_context *ctx)`

not sure the name will remain the same (I'm open to proposal :)